### ISSUE TITLE
Setting a custom background does not work

### ISSUE THREAD

**Issue by shiladitya-bits** [dmytrodanylyk/android-morphing-button#8]
I tried doing this:

<com.dd.morphingbutton.MorphingButton
            android:id="@+id/buttonSend"
            style="?android:attr/borderlessButtonStyle"
            android:background="@drawable/transact_button_send"
            android:layout_width="?android:attr/actionBarSize"
            android:layout_height="wrap_content"
            android:layout_weight="1"
            android:textColor="@color/white"
            android:text="SEND" />

The background still stays the default blue. What is the reason for this?

**Comment by shiladitya-bits** [comment]
I saw the ```initView()``` method in https://github.com/dmytrodanylyk/android-morphing-button/blob/master/library/src/main/java/com/dd/morphingbutton/MorphingButton.java

The corner radius and background drawables are hardcoded.

**Comment by OmarBizreh** [comment]
I have changed them in the code but somehow I still see the same color, @shiladitya-bits  did changing them from the code work for you?

**Comment by shiladitya-bits** [comment]
Nope. I ended up not using this library, and made my own set of animations for getting a morph effect.
